### ISSUE TITLE
a disabled test

### ISSUE THREAD

**Issue by DanGrayson** [Macaulay2/M2#824]
```
diff --git a/M2/Macaulay2/packages/EngineTests/Res.f4.m2 b/M2/Macaulay2/packages/EngineTests/Res.f4.m2
index 1824f3f67..99f34fce4 100644
--- a/M2/Macaulay2/packages/EngineTests/Res.f4.m2
+++ b/M2/Macaulay2/packages/EngineTests/Res.f4.m2
@@ -657,6 +657,11 @@ TEST ///
   I = AGR(3,12,15,101);
   I = AGR(3,11,15,101);
   R = ring I  
+
+  -- this test takes too much memory.
+  stderr << "--EngineTests/Res.f4.m2: *** Warning: bypassing a test that takes too much memory." << endl;
+  exit 0
+
   elapsedTime C = res(I, FastNonminimal=>true)
   gbTrace=2
   elapsedTime minimalBetti I
```

**Comment by DanGrayson** [comment]
and another one:
```
diff --git a/M2/Macaulay2/e/unit-tests/Makefile.in b/M2/Macaulay2/e/unit-tests/Makefile.in
index 8b142266b..f05c09403 100644
--- a/M2/Macaulay2/e/unit-tests/Makefile.in
+++ b/M2/Macaulay2/e/unit-tests/Makefile.in
@@ -33,12 +33,15 @@ all: $(UNITTEST_OBJECT_FILES) $(E_OBJECT_FILES)
 	echo @BUILTLIBS@
 	echo @LIBS@
 
-check: runtests
+# disabling a test temporarily, because it doesn't compile
+check:
+	@ echo "makefile: *** Warning : skipping temporarily disabled test in e/unit-test"
+# check: runtests
 
 fullCheck: $(UNITTEST_TARGET)
 	valgrind --track-origins=yes ./$(UNITTEST_TARGET)
 
-$(UNITTEST_TARGET) : gtest-include.o $(UNITTEST_OBJECT_FILES) $(E_OBJECT_FILES)
+$(UNITTEST_TARGET) : gtest-include.o $(UNITTEST_OBJECT_FILES) $(E_OBJECT_FILES) ../../system/supervisor.o
 	@CXX@ $(LDFLAGS) $^ $(LOADLIBES) -o $@
 
 runtests: $(UNITTEST_TARGET)
```

**Comment by mikestillman** [comment]
Fixed in commit 9aee9d4a2.  The unit tests are now run during `make check`.
